### PR TITLE
Fall 2020 Updates

### PR DESCRIPTION
**Pull request by sarahmthomas**
- switched out provided if-else example for case-switch example

### CODE REVIEW

File: src/Test1.java [hunk 1]
```diff
@@ -1,35 +1,37 @@
...
+                    result = num1 / num2;
+                }
+                break;
         }
-        else {
-            if(num2 == 0) {
-                System.out.println("Error: divide by zero");
-            }
-            else {
-                result = num1 / num2;
```
Comment (cindydhy) [inline src/Test1.java:25]: Can do a type conversion so it's not integer division?

Comment (sarahmthomas) [reply src/Test1.java:25]: good catch will do!

File: README.md [hunk 2]
```diff
@@ -87,13 +88,13 @@ Your new pull request can be found in the pull request tab. After clicking on it
...
 
 Immediately following the line of `=` is the version of the change from the *base* branch, and immediately following this line is a line that looks like: `>>>>>>> master`. 
 
-To fix the merge conflict, you need to remove the version of the change that you do not want, and keep the version that you do want. Right now, we want the version that uses case switch statements to implement the calculator functions properly. You also need to remove the line markers that git added: `<<<<<<< useCaseSwitch`, `=======`, and `>>>>>>> master`. 
+To fix the merge conflict, you need to remove the version of the change that you do not want, and keep the version that you do want. Right now, we want the version that uses case switch statements to implement the calculator functions properly. You also need to remove the line markers that git added: `<<<<<<< useIfElse`, `=======`, and `>>>>>>> master`. 
 
 Once you have that done, click the button on the top right that says `mark as resolved`. This will get you a green button that says `Commit merge`. Click this to commit this merge and solve your merge conflict!
```
Comment (sarahmthomas) [inline README.md:100]: Before merging, be sure to add a rubric here

File: src/Test1.java (hunk 1 above)
Comment (cindydhy) [reply src/Test1.java:25]: Cool!

**Pull request merged**